src/modules/admin/components/Permissions/index.ts

```diff
@@ -1,3 +1,3 @@
-export { default as ColonyPermissionsEditDialog } from './ColonyPermissionsEditDialog';
+export { default as PermissionManagementDialog } from '../../../dashboard/components/PermissionManagementDialog';
```

Comment:
`admin` doesn't need this export anymore, as there's no more permissions in the `admin` module